data-lineage/src/main/java/com/wipro/ats/bdre/lineage/LineageMain.java

```diff
@@ -604,7 +619,7 @@ private void handleTokSubQuery(ASTNode nextNodeOfFrom) {
 //		return ++nextId;
 //	}
 
-	private void printDot(Tree ct) {
+	/*private void printDot(Tree ct) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)